packages/grid/test/extension.test.js

```diff
@@ -1,5 +1,6 @@
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync } from '@vaadin/testing-helpers';
+import '../src/vaadin-grid.js';
```

Comment:
Could use `import { Grid } from '../src/vaadin-grid.js';` as we no longer need `customElements.get()`.

Reply:
UPD: same can be also done in `sorting.test.js` where we call static methods on the `Grid` class.